app/assets/javascripts/modules/media_viewer.js

```diff
@@ -146,11 +147,22 @@
       return hlsSource.prop('src').indexOf('/mp3:') >= 0;
     }
 
+    function ms10Browser() {
+      var fingerprinter = new ClientJS();
```

Comment:
I keep wanting to thumbsup this PR, but I don't know if y'all agreed on punting `/*global ClientJS */` or not.